### ISSUE TITLE
508-defect-3 [COGNITION]: Links should not be styled as other buttons on the page

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#17767]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a keyboard user, I expect to be able to use `space` to take action on a button. As a sighted user, I expect links to be visually styled as links so that I know I can open a link in a new tab.

## Details

On the review details page, the "Edit on Profile" link shares the same visual style as the other "Edit" buttons on the page. This causes several issues:
- It breaks the consistency of the experience.
- It's materially dishonest.
- Keyboard users may become frustrated attempting to use `space` the same way they are able to use `space` on the other edit buttons OR, they may confuse the rest of the edit buttons on the site to not register `space` .
- Sighted users will not know they can open their profile in a new tab.

## Acceptance Criteria

- [ ] Style the link as a link

## Steps to Recreate

1. Reach the review page `https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/review-and-submit` following the second test case [HLR - Same Office - No Conf](https://dsvavsp.testrail.io/index.php?/cases/view/91&group_by=cases:section_id&group_order=asc&group_id=18)
2. Tab to the link (styled as a button) "Edit on Profile" and confirm `space` does not work.

## Solution

Style the link as a link.

## WCAG or Vendor Guidance (optional)

* [WCAG 2.0 SC 3.2.4: Consistent Identification AA](https://www.w3.org/WAI/WCAG21/Understanding/consistent-identification.html): Components that have the same functionality within a set of Web pages are identified consistently.
* [Adam Silver: But sometimes buttons look like links](https://adamsilver.io/articles/but-sometimes-buttons-look-like-links/)

## Screenshots or Trace Logs

<img width="869" alt="Screen Shot 2020-12-20 at 5 48 21 PM" src="https://user-images.githubusercontent.com/14154792/102726485-35aacb00-42ed-11eb-8b80-c3dba1c3238c.png">

**Comment by Mottie** [comment]
This has been fixed in https://github.com/department-of-veterans-affairs/vets-website/pull/15442 and can be verified in staging.

**Comment by joshkimux** [comment]
<img width="692" alt="Screen Shot 2021-01-04 at 12 27 38 PM" src="https://user-images.githubusercontent.com/14154792/103561933-48de9e80-4e88-11eb-8026-1a6c7dc7e508.png">

Confirmed this has been fixed! Closing it out
